source/group_eq_bot/driver/local/reader.py

```diff
@@ -0,0 +1,49 @@
+from dataclasses import dataclass
+from typing import Union
+from pathlib import Path
+from io import BufferedReader
+
+from driver.local.audio.base import BaseAudioReader
+from driver.local.text.base import BaseTextReader
+from driver.models.supported_extensions import  SupportedFilesExtensions
+
+
+@dataclass
+class Reader:
+    path: Path  # full path to a particular file to open
```

Comment:
I'd suggest explicitly mark the `path` has to be initialised 
 
class Reader:
     path: Path  = field(init=True)

Reply:
`self.path` has to be initialised in a special way in post init.
So it remains as it was